src/command-line/uninstall.js

```diff
@@ -24,14 +24,20 @@ program
 		const packagesPath = Helper.getPackagesPath();
 		const packagesParent = path.dirname(packagesPath);
 		const packagesConfig = path.join(packagesParent, "package.json");
-		const packageWasNotInstalled = `${colors.green(packageName)} was not installed.`;
+		const packageWasNotInstalled = `${colors.green(packageName)} is not installed.`;
 
 		if (!fs.existsSync(packagesConfig)) {
 			log.warn(packageWasNotInstalled);
 			process.exit(1);
 		}
 
-		const npm = process.platform === "win32" ? "npm.cmd" : "npm";
+		const packages = JSON.parse(fs.readFileSync(packagesConfig, "utf-8"));
+
+		if (!packages.dependencies || !packages.dependencies.hasOwnProperty(packageName)) {
```

Comment:
So we are storing the plugins in package.json now?

Reply:
Yep, this is already in master due to npm bugginess.